loader/generated/vk_loader_extensions.c

```diff
@@ -10005,7 +10064,7 @@ PFN_vkVoidFunction get_extension_device_proc_terminator(struct loader_device *de
     // ---- VK_EXT_full_screen_exclusive extension commands
     if (!strcmp(name, "GetDeviceGroupSurfacePresentModes2EXT")) {
         *found_name = true;
-        return dev->extensions.ext_full_screen_exclusive_enabled && dev->extensions.khr_device_group_enabled ?
+        return dev->extensions.ext_full_screen_exclusive_enabled ?
```

Comment:
Same here

Reply:
I believe this is actually a 'more correct' enablement exprespression, because `vkGetDeviceGroupSurfacePresentModes2EXT` is available if either 1.1 is enabled or the device group extension is enabled. The spec is not very good in this regard. And I wonder if this should be an `or` instead of `and` but I digress